thoth/storages/graph/postgres.py

```diff
@@ -315,7 +316,7 @@ def connect(self):
         try:
             if not self.is_schema_up2date():
                 _LOGGER.debug("Database adapter connected, database is initialized")
-        except DatabaseNotInitialized as exc:
+        except Exception as exc:
```

Comment:
why this has changed?

Reply:
To be able to run locally on macOS. Will make as it was before sending to prod.